recurring_contract/models/recurring_contract.py

```diff
@@ -88,9 +88,18 @@ class RecurringContract(models.Model):
     company_id = fields.Many2one(
         'res.company',
         'Company',
+        # Show selection of all companies except Norden (id = 1)
+        domain="[('id', '!=', 1)]",
```

Comment:
It's really a bad idea to hardcode this here. When we will use it for Switzerland, if our company has id 1 we won't be able to set it correctly! Please move this code in a nordic-specific module, for instance `compassion_nordic_accounting`

Reply:
Thank you very much for your review, I moved this code statement to the compassion_nordic_accounting module 
[Related PR compassion-nordic 9](https://github.com/CompassionCH/compassion-nordic/pull/19)